### ISSUE TITLE
I just installed the 1.2 version, and there is no reset button. Please help.

### ISSUE THREAD

**Issue by raitan2008** [PhysicalAddons/physical-starlight-and-atmosphere#13]
**Describe the bug**
[Replace this with a clear and concise description of what the issue is about.]

**General information**
- Blender version: [e.g. 2.8.2]
- Engine: [e.g. Evee or Cycles]
- Operating System: [e.g. Windows 10, 	macOS(Catalina), etc.]
- Other info that could be useful: [e.g. CPU, GPU, etc.]: 

**To Reproduce**
[Replace this with a steps to reproduce the behavior]

**Screenshots**
[If applicable, add screenshots to help explain your problem.]

**Comment by karlisup** [comment]
Hi @raitan2008,
Have you tried to enable and disable the plugin in the `Preferences` panel? This may happen when upgrading to a newer version as the previous version is somehow interfering. Check also our [Troubleshooting page](https://starlight-manual.readthedocs.io/en/latest/installation/#physical-starlight-and-atmosphere-installation-instructions) in the documentation.

**Comment by raitan2008** [comment]
What was happening is that the old version was still in the N panel. Once I found the new version, the reset button was there. I also emailed you on another problem. In 2.83 the colors are all off. The sky is a murky blue and the sun is a dull orange. If I save a file in 2.82, and then open it in 2.83, then the sky colors and sun colors are fantastic again. Is there a fix for this? Using a RTX 1080 ti with a Ryzen 5 3600.
